islandora_find_replace.module

```diff
@@ -293,12 +305,17 @@ function islandora_find_replace_string_replace($search, $replace, $content) {
  * @return array
  *   An array of SPARQL results.
  */
-function islandora_find_replace_query($model, $collection = FALSE, $date_property = FALSE, $date_from = FALSE, $date_to = FALSE) {
+function islandora_find_replace_query($model, $namespace = FALSE, $collection = FALSE, $date_property = FALSE, $date_from = FALSE, $date_to = FALSE) {
```

Comment:
Could the new argument be added at the end? to make sure anyone else using this function in a non-standard way at least can have access to the original functionality?

Reply:
addressed in latest commit

Reply:
nice!